rcloud.support/R/zzz.R

```diff
@@ -7,4 +7,14 @@
              tryCatch(getNativeSymbolInfo("Rserve_oc_register"),
                       ## we cannot make this an error, because R attempts to try-load the package
                       error=function(e) if (is.null(getOption("rcs.silence.loadcheck"))) warning("WARNING: rcloud.support must be loaded by an Rserve instance, not stand-alone R!")))
+
+  printjs <<- rcloud.install.js.module("rcloud",
+                "(function() {
+                    return {
+                      console: function(message, k) {
+                        console.log(message)
+                      }
+                    }
+                  })()",
+                TRUE)
```

Comment:
We have `rcloud.session.log` for this (which prints to the session pane if available), so let's remove this.